docs/cloud/pouta/adding-images.md

```diff
@@ -173,6 +173,128 @@ you need to add the line "*acpiphp*" to */etc/modules*. For other
+create a temporary virtual machine that we will use to convert 
+myVmSnapshot. The virtual machine should have enough space to host 
+myVmSnapshot and its compact version at the same time. Since the 
+compact version will be smaller or of equal size of myVmSnapshot, 
+a safe choice is to select a flavor that is capable of hosting two times 
+the size of myVmSnapshot. For example, if myVmSnapshot has a size of 
+80GB, a suitable flavor for the auxiliary virtual machine is io.160GB 
+because it has 160 GB of ephemeral storage. The operating system can be 
+CentOS-7, for example.
+
+    openstack server create --flavor <flavor> \
+    --image <image uuid> \
+    --key-name <key name> \
+    --nic net-id=<name of network> \
+    --security-group default \
+    --security-group <additional security group> snapshotConverter
+
+The only additional requirements for the setting up of the virtual 
+machine are i) attaching a public floating IP, and ii) enabling SSH, so that 
+we can actually log into the virtual machine.
+
+2) Once the virtual machine is up and running, we copy the OpenStack 
+RC File v3 for accessing to cPouta/ePouta in the virtual machine. If you 
+do not have such file yet, please refer to [this 
+guide](https://docs.csc.fi/cloud/pouta/install-client/#configure-your-terminal-environment-for-openstack) 
+to obtain a copy.
+
+    scp <project_name_here>-openrc.sh cloud-user@<floating_ip>:/home/cloud-user/
+
+Login into the virtual machine and use the file to load your 
+credentials.
+
+    source <project_name_here>-openrc.sh
+
+3) In order to host the image obtained from the snapshot, we need to 
+initialize properly the ephemeral storage. To do so, please refer to [our 
+guide](https://docs.csc.fi/cloud/pouta/ephemeral-storage/). After this 
+step, we assume the ephemeral disk is mounted in _/mnt_.
+
+4) Next, we need to equip the virtual machine with some basic tools we 
+will need.
+
+    sudo yum install python3 python3-virtualenv screen qemu-img
+
+We create a python-3 virtual environment, which we will use to interact 
+with cPouta/ePouta, and we enter in it.
+
+    virtualenv-3 env
+    source env/bin/activate
+
+Now we install the actual tools we need to talk with cPouta/ePouta.
+
+    pip install python-openstackclient==3.11.0 openstacksdk==0.9.17 os-client-config==1.27.0 osc-lib==1.6.0
+
+5) Next, we download the image we obtained taking the snapshot. Move to 
+the ephemeral storage directory.
+
+    cd /mnt
+
+Though it is not required, at this point we recommend opening a _screen_ 
+session, which allows to keep a process running in background, i.e., 
+without the need of waiting for its completion before closing the 
+terminal.
+
+    screen -S converter
+
+We now issue the command to download the image obtained from the 
+snapshot.
+
+    openstack image save --file myVmSnapshotRaw.raw <id_of_myVmSnapshot>
+
+Given the size of the image, the process will take few minutes. We can 
```

Comment:
If it's only a few minutes, IMO screen is overkill. Have you encountered some scenario where it takes much more time? Then of course it makes sense.